docs/guides/hedgehog-engine/rangers/Model Imports/ModelIMP.md

```diff
@@ -0,0 +1,40 @@
+---
+description: Model Importing guide for Sonic Frontiers
+---
+# Model Importing
+
+!!! info
+    This guide uses the following tools:
+
+    - [HedgeArcPack](/tools/hedgehog-engine/common/files/#hedgearcpack){ target="_blank"}
+    -Blender
+    -ModelConverter.exe
+    -ModelFBX
+    -.modelImporter add-on for Blender
+
+# Extracting Models
+
+### Creating and Extracting the Pac File
+Firstly, create a Pac file named after the character and proceed to extract the contents of the pack utilizing the HedgeArcPack tool.
+### Converting to FBX and Importing into Blender
+Next, convert the `chr_character.model` file into the FBX format using the ModelFBX tool. Please note that certain models may not necessitate conversion into FBX format. 
+If you don't need to use the FBX format, use the HedgeNeedle tool, extract the various levels of detail from the `.model` file, and rename `chr_character.0.model` to `chr_character`. 
+Subsequently, utilize the model importer add-on to import the model into Blender. 
+### Rigging and Vertex Count
+After the model is imported into Blender, introduce your own model that is intended to be utilized in-game, and proceed to rig it to the original model's skeleton. 
+It is crucial to ensure that the vertex count of your model does not surpass the maximum requirements of the model converter, which is approximately 21,000 vertices per object. 
+### Assigning Weights and Resources
+As weights primarily refer to Blender rather than the Hedgehog Engine itself, you should use the following resources as they may be useful in learning how to manage weights:
+
+[Weight Transfer Guide](https://youtu.be/bR_Vke__voU)
+[Weight Painting Guide](https://youtu.be/4fICQmBEt4Y)
+### Rigging and Exporting the Model
+Once the character model has been rigged, select the model and the original skeleton, and proceed to export them as an FBX file. In case the original model is not deleted, choose 'selected objects only' in the export settings. 
+Although the other export settings are generally insignificant, it may be necessary to modify the scale to 0.01. 
+### Converting to .model and Creating Pac File
+Utilizing the `ModelConverter` tool, transform the FBX file into a .model file.
+Subsequently, place the new .model file into the previously extracted Pac folder. 
+To generate a new `.pac` file, drop the folder onto `HedgeArcPack`. The Pac file may then be used in your mod to observe the changes made. 
+### Materials and Textures
+For assistance with preparing materials and texture for Sonic Frontiers, refer to the `Advanced Material Editing Tutorial`.

```

Comment:
Make the refered guide a link instead, so that the user can click and go to that guide